.github/workflows/lintrunner_ci.yml

```diff
@@ -0,0 +1,123 @@
+name: Lintrunner CI
+
+on:
+  pull_request:
```

Comment:
We might want to limit this CI to just when things in the `lintrunner` folder get edited